src/Skia/Avalonia.Skia/GlyphTypefaceImpl.cs

```diff
@@ -24,40 +23,31 @@ public GlyphTypefaceImpl(SKTypeface typeface)
 
             Font.SetFunctionsOpenType();
 
-            Font.GetScale(out var xScale, out _);
+            DesignEmHeight = (short)Typeface.UnitsPerEm;
 
-            DesignEmHeight = (short)xScale;
+            var metrics = Typeface.ToFont().Metrics;
 
-            if (!Font.TryGetHorizontalFontExtents(out var fontExtents))
-            {
-                Font.TryGetVerticalFontExtents(out fontExtents);
-            }
+            Ascent = (int)(metrics.Ascent / 12.0 * Typeface.UnitsPerEm);
```

Comment:
just wondering where the 12.0 magic number came from @Gillibald ?

Reply:
Because Skia does not expose the unscaled version. By default, we have a font size of 12

Reply:
> Because Skia does not expose the unscaled version. By default, we have a font size of 12.

mb would be better to extract it to separate variable and to give it name like defaultFontSize for more self-descriptiveness?